app/streaming/audio/renderers/sdlaud.cpp

```diff
@@ -102,7 +151,11 @@ bool SdlAudioRenderer::submitAudio(int bytesWritten)
 {
     // Our device may enter a permanent error status upon removal, so we need
     // to recreate the audio device to pick up the new default audio device.
+#if SDL_VERSION_ATLEAST(3, 0, 0)
+    if (SDL_AudioDevicePaused(SDL_GetAudioStreamDevice(m_AudioStream))) {
```

Comment:
Just realized we never added a stream-specific equivalent of SDL_AudioDevicePaused (whoops!), so for now at least, this part _does_ need to call SDL_GetAudioStreamDevice().